aten/src/ATen/native/quantized/TensorFactories.cpp

```diff
@@ -116,6 +116,7 @@ Tensor empty_quantized(
                               .merge_memory_format(memory_format);
 
   Tensor output;
+
```

Comment:
Can this change be reverted so that this file is not modified?

Reply:
ya, will do